### ISSUE TITLE
Parser is not streaming

### ISSUE THREAD

**Issue by emersion** [luxifer/ical#11]
The whole blob is loaded in memory: https://github.com/luxifer/ical/blob/master/parse.go#L71

While this isn't an issue for files with a single event, this might be an issue for big exports/archives.

**Comment by luxifer** [comment]
You're right. This wasn't an issue at the time I write this lib. Feel free to submit a PR to fix that.
